treeherder/log_parser/parsers.py

```diff
@@ -408,11 +408,17 @@ class ErrorParser(ParserBase):
 
     RE_ERR_1_MATCH = re.compile(r"^\d+:\d+:\d+ +(?:ERROR|CRITICAL|FATAL) - ")
 
+    # [task 2016-08-18T17:50:56.955523Z]
+    # We require a space inside the square brackets because lines beginning
+    # with "[taskcluster:error]" signify an error and must not be ignored.
```

Comment:
Doing detection based on the presence of a space (or not) seems kind of brittle to me.